.github/workflows/labeler.yml

```diff
@@ -0,0 +1,17 @@
+name: "Pull Request Labeler"
+on:
+- pull_request_target
+
+jobs:
+  labeler:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        repository: "ggerganov/llama.cpp"
```

Comment:
What's this? Why are we checking out ggerganov's repo?

Reply:
Ah, overlooked this when porting this over from there. Corrected to Mozilla-Ocho/llamafile